src/lib/day-calendar/day-calendar.component.spec.ts

```diff
@@ -138,4 +138,42 @@ describe('Component: DayCalendarComponent', () => {
     component.goToCurrent();
     expect(component.onGoToCurrent.emit).toHaveBeenCalled();
   });
+
+  it('should not display secondary navigation button when not asked to', () => {
+    const currentDate = moment();
+    component.currentDateView = currentDate; // updates showLeftNav && showLeftSecondaryNav
```

Comment:
can you please put the comment above the code (same for all comments)

Reply:
Done